CoinDock-api/app/Models/V1/Coin.php

```diff
@@ -22,6 +22,22 @@ public function wallets(){
         return $this->hasMany(Wallet::class);
     }
 
+    public function acceptedAssets(){
+        $acceptedAssets = $this::whereStatus(1)->get();
```

Comment:
Are we using acceptedAssets() anywhere? I didn't see the usage of the function. 

If it's an unused function then please remove it. Otherwise, ignore this comment.

Reply:
done